### PR TITLE
Github status should be pending when build is running/queued

### PR DESCRIPTION
**Pull request by d2lam**
Right now the status is failure when the build is queued/running, which is not correct.

### CODE REVIEW

File: lib/build.js [hunk 1]
```diff
@@ -229,12 +229,20 @@ class BuildModel extends BaseModel {
...
                         pipeline.admin
                         // update github
-                            .then(admin => updateSCM({
-                                user: admin,
-                                scmUrl: pipeline.scmUrl,
-                                sha: this.sha,
-                                state: (this.status === 'SUCCESS') ? 'success' : 'failure'
-                            }))
+                            .then(admin => {
+                                const stateMap = {
```
Comment (FenrirUnbound) [inline lib/build.js:233]: This is basically a map of constants. Define them at the top of the file

Comment (d2lam) [inline lib/build.js:233]: This will probably be removed after the git scm is done anyway

Comment (FenrirUnbound) [inline lib/build.js:233]: "Probably"? Or "Will"?

Comment (d2lam) [inline lib/build.js:233]: will. that's what Noah is working on

Comment (FenrirUnbound) [inline lib/build.js:233]: 👍 👌

**Comment by FenrirUnbound** [comment]
👍 

Looks good to me

uxjywqc

**Comment by nkatzman** [comment]
👍

**Pull request merged**